packages/fxa-customs-server/lib/config/config.js

```diff
@@ -475,61 +475,61 @@ module.exports = function (fs, path, url, convict) {
           },
         },
       },
-      recoveryPhoneCreateCodeRules: {
+      recoveryPhoneSendSetupCodeRules: {
         actions: {
           doc:
             'Array of actions that this rule should be applied to. A user should only be able to initiate ' +
-            'a recovery phone setup X times a day.',
-          default: ['recoveryPhoneCreate'],
+            'a recovery phone setup X times a day by sending a setup code.',
+          default: ['recoveryPhoneSendSetupCode'],
           format: Array,
         },
         limits: {
           max: {
             doc: 'max actions during `period` that can occur before rate limit is applied',
             format: 'nat',
             default: 9,
-            env: 'RECOVERY_PHONE_SETUP_CODE_RULE_MAX',
+            env: 'RECOVERY_PHONE_SEND_SETUP_CODE_RULE_MAX',
           },
           periodMs: {
             doc: 'period needed before rate limit is reset',
             format: 'duration',
             default: '1 day',
-            env: 'RECOVERY_PHONE_SETUP_CODE_RULE_PERIOD_MS',
+            env: 'RECOVERY_PHONE_SEND_SETUP_CODE_RULE_PERIOD_MS',
           },
           rateLimitIntervalMs: {
             doc: 'how long rate limit is applied',
             format: 'duration',
-            default: '2 hours',
-            env: 'RECOVERY_PHONE_SETUP_CODE_RULE_LIMIT_INTERVAL_MS',
+            default: '24 hours',
+            env: 'RECOVERY_PHONE_SEND_SETUP_CODE_RULE_LIMIT_INTERVAL_MS',
           },
         },
       },
-      recoveryPhoneConfirmCodeRules: {
+      recoveryPhoneSendSigninCodeRules: {
         actions: {
           doc:
             'Array of actions that this rule should be applied to. A user should only be able to initiate ' +
-            'a recovery phone confirm X times a day.',
-          default: ['recoveryPhoneConfirmCode'],
+            'a recovery phone signin X times a day by sending an sms code.',
+          default: ['recoveryPhoneSendSigninCode'],
           format: Array,
         },
         limits: {
           max: {
             doc: 'max actions during `period` that can occur before rate limit is applied',
             format: 'nat',
             default: 6,
-            env: 'RECOVERY_PHONE_CONFIRM_CODE_RULE_MAX',
+            env: 'RECOVERY_PHONE_SEND_SIGNIN_CODE_RULE_MAX',
           },
           periodMs: {
             doc: 'period needed before rate limit is reset',
             format: 'duration',
             default: '1 day',
-            env: 'RECOVERY_PHONE_CONFIRM_CODE_RULE_PERIOD_MS',
+            env: 'RECOVERY_PHONE_SEND_SIGNIN_CODE_RULE_PERIOD_MS',
           },
           rateLimitIntervalMs: {
             doc: 'how long rate limit is applied',
             format: 'duration',
-            default: '2 hours',
-            env: 'RECOVERY_PHONE_CONFIRM_CODE_RULE_LIMIT_INTERVAL_MS',
+            default: '24 hours',
```

Comment:
@vbudhram I changed this so that it's the same as the duration. This is how most of our limits work, and results in a better end user message. ie Try again in 1 day as opposed to try again in 2hours, at which point they'd just be blocked again. We need to double check how the blocks work here though. Can you help validate that this won't block other user interactions on this email or IP?  I'll also try to validate this, but want to be extra sure it doesn't cause problems.

If it turns out this does block other user interactions, we might need to make and adjustment to the customs rules for this... because blocking someone for 24 hours because they've sent too many SMS would be a bad UX.

Reply:
@vbudhram Just following up here. I retested several times locally, and have not seen blocking on other actions.

Reply:
@vbudhram Just found the issue after posting the above comment. There was a `customs.check` call, that should have been `customs.checkAuthenticated` call in `RecoveryPhoneHandler -> sendCode`. I think we are good to go now. :+1: